src/webextension/widgets/link.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import PropTypes from "prop-types";
+import React from "react";
+
+import styles from "./link.css";
+
+export function ExternalLink({onClick, children}) {
+  return (
+    <a className={styles.external} onClick={onClick}>{children}</a>
```

Comment:
Two small things. Let users of this component pass:

1. Extra `props` and apply them to the `<a>` tag: `<a {...props}>`
2. A `className` to add styles for a specific use of this component. This is the snippet I use elsewhere: `const finalClassName = `${styles.external} ${className}`.trimRight();`